entrypoint.sh

```diff
@@ -4,14 +4,17 @@ set -e
 # If we are called with default arguments, invoke SickGear with datadir and
 # setup the environment
 if [ "$1" = 'SickBeard.py' ]; then
+  APP_DATA_UIDGID="${APP_UID:=0}:${APP_GID:=0}"
   if [ ! -f ${APP_DATA}/config.ini ]; then
-    cp /template/config.ini ${APP_DATA}/
+    exec gosu ${APP_DATA_UIDGID} cp /template/config.ini ${APP_DATA}/
```

Comment:
Hmm.. this `exec` will actually prevent from rest of the script happening. `exec` command replaces the current script with the command about to be run. Simply removing the `exec` will fix the behavior